recipes/pkgconfig/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pkgconfig" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "4de443a62f4b83d883a7b3895d4d4d3ca1934383ebdbeb0f2e6ab126085d812a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
```

Comment:
Would suggest this be `{{ name }}-{{ version }}.tar.gz`.

Note: It doesn't matter what the actual file downloaded is named as `conda-build` will rename it with the `fn` provided. Doing it as proposed will be helpful for people building offline, but does not strictly matter for conda-forge.